app/template/page.tsx

```diff
@@ -1,29 +1,118 @@
 'use client';
 
 import React from 'react';
-import {Link} from "react-aria-components";
+import {
+  Breadcrumb,
+  Breadcrumbs,
+  Button,
+  FieldError,
+  Input,
+  Label,
+  Link,
+  SearchField,
+  Text
+} from "react-aria-components";
+import PageHeader from "@/components/PageHeader";
+import TemplateListItem from "@/components/TemplateListItem";
 
 
 const TemplateListPage: React.FC = () => {
+
+  const templates = [
+    {
+      title: 'Arctic Data Center: NSF Polar Programs',
+      link: '/template/tpl_2525235',
+      content: (
+        <div>
+          <p>National Science Foundation (nsf.gov)</p>
+          <p>Last updated: 04-01-2024</p>
+        </div>
+      ),
+      funder: 'National Science Foundation',
+      lastUpdated: '04-01-2024',
+      publishStatus: 'Published',
+      defaultExpanded: false,
+    },
+    {
+      title: 'NSF Polar Expeditions',
+      link: '/template/tpl_Ad525',
+      content: (
+        <div>
+          <p>National Science Foundation (nsf.gov)</p>
+          <p>Last updated: 06-15-2024</p>
+          <p>Status: Unpublished</p>
+        </div>
+      ),
+      funder: 'National Science Foundation',
+      lastUpdated: '06-15-2024',
+      publishStatus: 'Unpublished',
+      defaultExpanded: false,
+    },
+    {
+      title: 'NSF: McMurdo Station (Antarctic)',
+      link: '/template/tpl_Ad525',
+      content: (
+        <div>
+          <p>National Science Foundation (nsf.gov)</p>
+          <p>Last updated: 09-21-2024</p>
+        </div>
+      ),
+      funder: 'National Science Foundation',
+      lastUpdated: '09-21-2024',
+      publishStatus: 'Published',
+      defaultExpanded: false,
+    }
+  ];
+
+
+
+
   return (
     <>
-      <h1>Templates</h1>
 
-      <Link href="/template/create">Create</Link>
+      <PageHeader
+        title="Templates"
```

Comment:
I'm still wondering whether it would be more intuitive to be able to click on the whole "Arctic Data Center" card, for instance, instead of having to click on "Update" link.

Reply:
I don't really disagree. My thinking of why I did it that way was kind of accessibility thinking

Focus: Keyboard and screen reader users may struggle to navigate if the whole card is clickable - it makes it a bit "weird" all things are focuable as it becomes a "block element"

So screen readers might misinterpret the card's purpose, especially with nested buttons like "Expand."

Next conflicting actions: Inner elements (e.g., "Expand" button) - having full card clickable then create issues with our "secondary" action the expand button and not just screen readers but people with impaired motor functions, it might require a more "specific" selection or click - so I had some concerns over that 

Then additionally - more of a side thing  not sure what the best practise on a focus state of something like this is?

Reply:
Thanks for the discussion on this Fraser. I've seen websites where whole cards are clickable and focusable, maybe that's why I keep expecting the whole card to be clickable. I guess we can leave it as is, and see how it does in further testing. Maybe it's just me :)